src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -300,19 +301,22 @@ export default class AircraftCommander {
      * @return {array} [success of operation, readback]
      */
     runHold(aircraft, data) {
-        const turnDirection = data[0];
-        const legLength = data[1];
-        const fixName = data[2];
+        const [turnDirection, legLength, fixName, radial] = data;
         const fixModel = NavigationLibrary.findFixByName(fixName);
 
         if (!fixModel) {
             return [false, `unable to hold at unknown fix ${fixName}`];
         }
 
+        // Radial is given as the outbound course, so it needs to be inverted
+        const inboundHeading = radial === null ?
+            fixModel.positionModel.bearingFromPosition(aircraft.positionModel) :
+            radians_normalize(degreesToRadians(radial) + Math.PI);
```

Comment:
```
let inhoundHeading = fixModel.positionModel.bearingFromPosition(aircraft.positionModel);

if (radial) {
    inhoundHeading = radians_normalize(degreesToRadians(radial) + Math.PI);
}
```

I'm not saying this is any better or worse, your solution works fine, we just tend to avoid ternaries unless they're small and clearly advantageous.

Reply:
@erikquinn fun fact: adding `js` after the three bat ticks of a code block will give you syntax highlighting.  I've edited your comment to use that 😉

Reply:
I first went with that, but opted for the ternary so I could declare it const. Plus it doesn't have to calculate bearingFromPosition unless needed.

Because you never know if you'll be requesting holds in a tight loop :grin: